lerna.json

```diff
@@ -2,7 +2,7 @@
     "npmClient": "npm",
     "packages": ["packages/*"],
     "useWorkspaces": true,
-    "version": "independent",
+    "version": "1.3.0",
```

Comment:
Shouldn't this be `2.0.0`?

Reply:
no this will be generated on first release with lerna automaticly